model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -61,3 +61,23 @@ data class ScannerConfiguration(
      */
     val storageWriters: List<String>? = null
 )
+
+/**
+ * A class defining scanner-specific option.
+ *
+ * In the global [ScannerConfiguration], the _options_ map allows assigning an instance of this class to each
+ * scanner. That way, this scanner can be configured in a special way. The options consist of a part that is common to
+ * all scanners, and a generic map of properties to be evaluated by specific scanner implementations.
+ */
+data class ScannerOptions(
```

Comment:
Similarly, this class is not really scanner-specific, and could be moved as e.g. `ToolOptions` to its own file.

Reply:
This is no longer the case when the _compatibility_ object becomes scanner-specific.

Reply:
But what's the rationale for putting `ScannerCompatibilityConfiguration` into its own file, but not `ScannerOptions`?